src/components/search/SearchResults.jsx

```diff
@@ -11,17 +11,17 @@ import SearchError from './SearchError';
 import { isDefinedAndNotNull } from '../../utils/common';
 import { NUM_RESULTS_PER_PAGE } from './data/constants';
 import {
-  useRefinementsFromQueryParams,
   useNbHitsFromSearchResults,
 } from './data/hooks';
+import { SearchContext } from './SearchContext';
 
 const SearchResults = ({
   searchResults,
   searchState,
   isSearchStalled,
   error,
 }) => {
-  const refinementsFromQueryParams = useRefinementsFromQueryParams();
```

Comment:
Double check that nothing is modifying `refinementsFromQueryParams` in this file.